### PR TITLE
Add long running tests for serverless

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**
Add long running tests for serverless.

Changes proposed in this pull request:

- Serverless long running test added

**Related issue(s)**

See also: https://github.com/kyma-project/kyma/issues/8730

### CODE REVIEW

File: resources/serverless/templates/tests/long-test.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            - "/bin/bash"
+          args:
+            - "-c"
+            - |
+              echo 'TESTING has started'
+              sleep 20
+
+              # test start
+
+              curl -XGET http://{{ .Values.tests.long.name }}.{{ .Values.tests.long.namespace }} \
```
Comment (michal-hudy) [inline resources/serverless/templates/tests/long-test.yaml:68]: Please also check if Function CR has all conditions set to `true`, as status can be invalid.

Comment (magicmatatjahu) [reply resources/serverless/templates/tests/long-test.yaml:68]: Please also check that response is `OK` :)

Comment (m00g3n) [reply resources/serverless/templates/tests/long-test.yaml]: @magicmatatjahu The result code is enough, why you want to check the response body?

**Pull request merged**